src/Dynamo.All.sln

```diff
@@ -513,6 +513,10 @@ Global
 		{F7FD9395-35D5-474D-8AD1-9904817E70D3}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{F7FD9395-35D5-474D-8AD1-9904817E70D3}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{F7FD9395-35D5-474D-8AD1-9904817E70D3}.Release|Any CPU.Build.0 = Release|Any CPU
+		{676671D5-76B6-4E3B-9244-5323BF6B8EFE}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
what are these changes to the dynamo.sln?

Reply:
This is because the project TransformXML CSVTool was in the main solution but in the next commit was removed from the main solution and this project TransformXML CSVTool has its own solution on its location.

Reply:
please revert these changes then. Keep the PR as minimal as possible.

Reply:
Yes, I already reverted it.